src/crimson/os/seastore/seastore.cc

```diff
@@ -716,9 +722,15 @@ SeaStore::Shard::list_objects(CollectionRef ch,
 		  std::get<1>(ret) = std::get<1>(_ret);
 		  assert(limit >= next_objects.size());
 		  limit -= next_objects.size();
+		  LOG_PREFIX(SeaStore::list_objects);
+		  DEBUGT("got {} objects, left limit {}",
+		    t, next_objects.size(), limit);
+		  if (last && std::get<1>(ret) == pend) {
+		    std::get<1>(ret) = end;
```

Comment:
Let's add comments to `SeaStore::list_objects(c, start, end, limit)` in seastore.h:

/// std::get<1>(ret) returns end if and only if the listing has listed all the items within the range, otherwise it returns the next key to be listed.

Seems to me it infers the comment https://github.com/ceph/ceph/blob/main/src/os/ObjectStore.h#L678

Reply:
Yeah, will do:-)

Reply:
Done